docs/monthly-meeting/2022-09.md

```diff
@@ -0,0 +1,72 @@
+# Documentation Community Team Meeting (September 2022)
+
+- **Date:** 2022-09-12
+- **Time:** [21:00 UTC](https://arewemeetingyet.com/UTC/2022-09-12/21:00/Docs%20Meeting)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- **Discourse thread:** https://discuss.python.org/t/18590
+- **Calendar for future meetings:** (send your e-mail to Mariatta for a calendar invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [hackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+- Petr Viktorin / `@encukou`
+- Ezio Melotti / `@ezio-melotti`
+- Mariatta / `@mariatta`
+- Ned Batchelder / `@nedbat`
+- C.A.M. Gerlach / `@CAM-Gerlach`
+
+
+## Reports and celebrations
+
+- We will be publishing the Workshop videos to YouTube soon. Stay tuned for
+  announcement and emails.
+
+- Ee is back, so several things should get unblocked in the coming weeks,
+  like the YouTube uploads or docs preview on Netlify
+
+- What's New editing (**CAM**)
+  - Major PEP sized items all added except for PEP 670
+  - First set of copyedit PRs waiting on review/merge; please help!
+  - Refs from summary to body content will be PRed very soon
+  - Remaining items will be submitted as PRs soon(™)
```

Comment:
```suggestion
  - Remaining items will be submitted as PRs soon(™)
  - https://github.com/python/cpython/issues/95913
```

Reply:
Good point, but we should add this as an actual proper link, and it would be nice to link `#95914` too

```suggestion
  - Remaining items will be submitted as PRs soon(™)
  - See [python/cpython#95913](https://github.com/python/cpython/issues/95913) and [python/cpython#95914](https://github.com/python/cpython/issues/95914)
```